pkg/constant/k8s.go

```diff
@@ -75,3 +76,16 @@ const (
 	SpiderControllerElectorLockName = SpiderpoolController + "-" + resourcelock.LeasesResourceLock
 	QualifiedK8sObjNameFmt          = "[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*"
 )
+
+// subnet manager annotation and labels
+const (
+	SubnetManagerPrefix = SpiderpoolAPIGroup
+
+	AnnoSubnetManagerV4 = SubnetManagerPrefix + "/spider-subnet-v4"
```

Comment:
要统一风格，看看存量的都是什么风格，
AnnoNSDefautlV6Pool = AnnotationPre + "/defaultv6ippool"
建议如
AnnoSubnetManagerV4 = AnnotationPre + "/ipv4Subnet"

Reply:
Key names should be all lowercase, with words separated by dashes instead of camelCase
For instance, prefer foo.kubernetes.io/foo-bar over foo.kubernetes.io/fooBar, prefer desired-replicas over DesiredReplicas

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#label-selector-and-annotation-conventions
后期会提个pr，统一将存量的改成*-*格式